common/web/keyboard-processor/src/text/codes.ts

```diff
@@ -86,7 +86,7 @@ const Codes = {
     [')!@#$%^&*(',':+<_>?~', '{|}"']
   ],
 
-  isKnownOSKModifierKey(keyID: string): boolean {
+  isKeyNotCorrected(keyID: string): boolean {
```

Comment:
I changed the name for two reasons:

1. This highlights the limited uses of the method.
2. Given that the only use is to determine what keys we don't fat-finger correct, and that K_TAB + K_TABFWD are being included... well, K_TAB isn't a modifier key.